ibc_public/LabelsDb.py

```diff
@@ -0,0 +1,105 @@
+import json
+import sys
+import pandas as pd
+
+import os
+
+
+class LabelsDb:
+    """
+    Opens a .tsv files containing contrasts and cognitive labels, and allows
+    to retrieve information from it, as well as adding new labels to existing
+    contrasts
+
+    Attributes
+    ----------
+
+    path: str, Default None
+          Path to file
+
+    contrast_col: str, default 'contrast'
+                  Name of the column of your dataframe that contains contrast
+                  information
+
+    sep: str, default '\t'
+         Separator used in your file, to be passed to pandas
+
+    data: pd.DataFrame
+          DataFrame object containing all the information
+    """
+
+    config_path = os.path.join(os.path.dirname(__file__), 'config.json')
```

Comment:
What is this config_path ? Why do we need it ? Why isn't it part of the PR ?

Reply:
It was used to work outside of the public repo, not it is removed and it handles the path like other functions that depend on files present on the repo